tests/mocked/test_uptime.py

```diff
@@ -0,0 +1,224 @@
+import responses
+
+from pydo import Client
+
+
+@responses.activate
+def test_uptime_check_lists_get(mock_client: Client, mock_client_url):
+    """Mocks the uptime checkLinst operations"""
+
+    expected = {
+        "checks": [
+            {
+                "id": "5a4981aa-9653-4bd1-bef5-d6bff52042e4",
+                "name": "Landing page check",
+                "type": "https",
+                "target": "https://www.landingpage.com",
+                "regions": [
+                    "us_east",
+                    "eu_west"
+                ],
+                "enabled": True
+            }
+        ],
+        "links": {
+            "pages": {
+                "pages": {
+                    "first": "https://api.digitalocean.com/v2/account/keys?page=1",
+                    "prev": "https://api.digitalocean.com/v2/account/keys?page=2"
+                }
+            }
+        },
+        "meta": {
+            "total": 1
+        }
+    }
+
+    responses.add(
+        responses.GET,
+        f"{mock_client_url}/v2/uptime/checks",
+        json=expected,
+        status=200
+    )
+    get_res = mock_client.uptime.list_checks()
+    assert get_res == expected
+
+
+@responses.activate
+def test_check_state_get(mock_client: Client, mock_client_url):
```

Comment:
if you run make lint-tests, you'll see a bunch of errors asking:
tests/mocked/test_uptime.py:42:0: C0116: Missing function or method docstring (missing-function-docstring)
This is super easy fix, you'll just have to add "docstrings" under the function, as such:

```suggestion
def test_check_state_get(mock_client: Client, mock_client_url):
    """Mock Uptime Check State"""
```